eng/versioning/update_changelog.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import sys
+import os
+import logging
+import datetime
+import re
+
+CHANGE_LOG_FILE_NAME = "history.md"
```

Comment:
fyi @chidozieononiwu another place you will need to update when you rename these from history to changelog.